[DOC] address document issues [skip ci] (#9181)
NVIDIA/spark-rapids: docs/FAQ.md

that the RAPIDS Accelerator suits your needs, and file and issue or discussion if you have doubts
or would like expanded encryption support.

> ### What is the special requirement for running Rapids Accelerator with Spark on Ray?

Reviewer: ```suggestion
### Can the Rapids Accelerator work with Spark on Ray (RayDP)?
```

> RayDP provides simple APIs for running Spark on [Ray](https://github.com/ray-project/ray) and 
> integrating Spark with AI libraries, we need to make the following configuration changes to fix
> the compatibility issues with the RAPIDS Accelerator and Ray:

Reviewer: ```suggestion
[RayDP](https://github.com/oap-project/raydp) provides simple APIs for running Spark on
[Ray](https://github.com/ray-project/ray). In order to run the RAPIDS Accelerator with RayDP,
GPUs must be requested as `GPU` resources rather than `gpu` resources. This can be done
with the following changes to the typical GPU resource scheduling setup with Spark:
```

> * Changed the `gpuGpusResources.sh` script to use `GPU` instead of `gpu`
>   
> * Changed the `spark.executor.resource.gpu.*` configs to corresponding `spark.executor.resource.GPU.*` configs
>   
> * Changed `spark.task.resource.gpu.amount` to `spark.task.resource.GPU.amount`

Reviewer: ```suggestion
* Change the contents of the `getGpusResources.sh` script to use `GPU` instead of `gpu`  
* Change all of the configs that start with `spark.executor.resource.gpu.` to corresponding configs that start with `spark.executor.resource.GPU.`  
* Change `spark.task.resource.gpu.amount` to `spark.task.resource.GPU.amount`
```

> 

Reviewer: ```suggestion

Note that the RAPIDS Accelerator is not regularly tested against Spark on Ray.
```

> Note that the RAPIDS Accelerator is not regularly tested against Spark on Ray.
> ### I have more questions, where do I go?

Reviewer: ```suggestion
Note that the RAPIDS Accelerator is not regularly tested against Spark on Ray.

### I have more questions, where do I go?
```
